packages/analytics-ma-browser/README.md

```diff
@@ -0,0 +1,189 @@
+
+amplitude.init("YOUR_API_KEY_HERE");
+</script>
+```
+<!-- / OF README_SNIPPET_BLOCK -->
+
+## Usage
+
+### Initializing SDK
+
+Initialization is necessary before any instrumentation is done. The API key for your Amplitude project is required.
+
+```typescript
+amplitude.init(API_KEY);
+
+// Config web attribution and auto page view tracking
+amplitude.init(API_KEY, USER_ID, {
+  // Config web attribution
+  attribution: {
+    resetSessionOnNewCampaign: true;
+  },
+
+  // Enable auto page view tracking
+  trackPageViews: {
+    trackOn: 'attribution',
+  }
+});
+```
+
+### Tracking an Event
+
+Events represent how users interact with your application. For example, "Button Clicked" may be an action you want to note.
+
+```typescript
+import { track } from '@amplitude/analytics-browser';
+
+// Track a basic event
+track('Button Clicked');
+
+// Track events with additional properties
+const eventProperties = {
+  selectedColors: ['red', 'blue'],
+};
+track('Button Clicked', eventProperties);
+```
+
+### User Properties
+
+User properties help you understand your users at the time they performed some action within your app such as their device details, their preferences, or language.
+
+```typescript
+import { Identify, identify } from '@amplitude/analytics-browser';
```

Comment:
same here and for the rest

Reply:
not sure if we should have all of these examples or just send them to dev docs 🤔